cmake/Bond.cmake

```diff
@@ -129,6 +130,18 @@ function (add_bond_test test)
             WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
             COMMAND ${test})
     endif()
+
+    get_property (lastTestPort GLOBAL PROPERTY LAST_TEST_PORT)
+    if (NOT DEFINED lastTestPort)
+        set (lastTestPort 25189)
+    endif()
+    math(EXPR testPort1 "${lastTestPort} + 1")
+    math(EXPR testPort2 "${lastTestPort} + 2")
+    target_compile_definitions (${test} PRIVATE -DTEST_PORT_1=${testPort1})
```

Comment:
That is a fun fact.

Given that this function (and, really, cmake) has no way to express a structured map properly, I think I would prefer that the opaque strings we pass be the right opaque strings for the layer they're destined for. I prefer `-DMACRO`.